### ISSUE TITLE
currencyList.get is not a function

### ISSUE THREAD

**Issue by xXAvoraXx** [webplan-pro/currency-list#2]
After the new update the functions stopped working.

```
import currencyList from 'currency-list';

currencyList.get('USD');
```

TypeError: currencyList.get is not a function

**Comment by tuturis** [comment]
Thanks for the notification and sorry for the inconvenience. The webpack build that I changed while patching, did not respect default exports so I changed the build back

**Comment by tuturis** [comment]
Patched to 1.0.8 btw.
